File: D1-Morning/C#/D1-S1/P1-CreateAnObject/P1-CreateAnObject/Person.cs
namespace P1_CreateAnObject
{
    class Person
    {
        public string firstName;
        public string middleName;
        public string lastName;

        public string GetMyFullName()
        {
            return firstName + " " + middleName + " " + lastName;
        }

        public string GetFullReverseName()
        {
            string fullName = GetMyFullName();
            string reverseName = "";
            for (int index = fullName.Length - 1; index >= 0; index--)
            {
                reverseName += fullName[index];
            }
            return reverseName;
        }
    }
}

File: D1-Evening/D1-S4 - PHP/P13-SRP/P13-SRP-Tushar/Person.php
<?php


class Person
{
  private $firstName;
  private $middleName;
  private $lastName;
  private $fullName;


  public function getFirstName()
  {
    return $this->firstName;
  }


  public function setFirstName($firstName)
  {
    $this->firstName = $firstName;
  }


  public function getMiddleName()
  {
    return $this->middleName;
  }


  public function setMiddleName($middleName)
  {
    $this->middleName = $middleName;
  }


  public function getLastName()
  {
    return $this->lastName;
  }

  public function setLastName($lastName)
  {
    $this->lastName = $lastName;
  }

  public function getFullReverseName()
  {

    $fullName = $this->getMyFullName();
    $reverseName = "";
    for ($index = strlen($fullName) - 1; $index >= 0; $index--) {
      $reverseName = $reverseName . $fullName[$index];
    }
    return $reverseName;

  }

  public function getMyFullName()
  {
    $this->fullName = $this->firstName . " " . $this->middleName . " " . $this->lastName;
    return $this->fullName;
  }
}